docs/docs-beta/docs/guides/asset-sensors.md

```diff
@@ -68,22 +71,25 @@ stateDiagram-v2
     classDef userDefined fill: var(--theme-color-accent-lavendar)
 ```
 
-In this example, the `@asset_sensor` decorator allows you to define a custom evaluation function that returns a `RunRequest` object when the asset is materialized and certain metadata is present,
-otherwise it skips the run.
+In the following example, the `@asset_sensor` decorator defines a custom evaluation function that returns a `RunRequest` object when the asset is materialized and certain metadata is present, otherwise it skips the run.
```

Comment:
I found the OG sentence here a little odd (is it really the decorator that allows this? Is it because of `context` being used?) but I didn't know a better way to word this.